kyo-prelude/jvm/src/main/scala/kyo2/kernel/Platform.scala

```diff
@@ -0,0 +1,25 @@
+package kyo2.kernel
+
+import kyo2.kernel.Mode
+import scala.concurrent.ExecutionContext
+
+object Platform:
+
+    val mode: Mode =
```

Comment:
I've added this `Mode` mechanism so we can vary configurations based on it. For example, if the mode is `Debugging`, we could render a fancier trace. This is unused yet, though.